frontend/Community.css

```
@@ -600,93 +600,384 @@
+  margin-top: 1.7rem;
+  display: flex;
+  gap: 0.8rem;
+  overflow-x: auto;
+  padding-bottom: 6px;
 }
 
-.char-counter--ok {
-  background: #dcfce7;
-  color: #166534;
+.cat-tab {
+  border: none;
```

Stylelint (17.10.0)

[error] Duplicate property "border" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)